### ISSUE TITLE
Use app-specific data directory for internal files (required in Android 11)

### ISSUE THREAD

**Issue by Victor239** [foobnix/LibreraReader#681]
Librera Reader creates a new directory in shared storage which contains it's files. This is messy when there already exists an [app-specific data directory](https://developer.android.com/training/data-storage/app-specific) for it. Shared storage is for files the user should have direct access to e.g. the ebooks themselves. 

Additionally [this will be required](https://developer.android.com/about/versions/11/privacy/storage) anyway in Android 11 so it makes sense to do so now. 

**Ideally:**
Use app-specific data directory for all Librera internal files. Let the user configure a directory on shared storage where they keep fonts or other things that you want configurable, or allow them to add them into the internal file storage via a file picker.

**Comment by JJ-20** [comment]
Yes currently librera dosen't use it's own storage by default which isn't a user friendly decision for android 11+, please consider this.
